packages/ti_abusech/elasticsearch/transform/latest_url/fields/fields.yml

```
@@ -0,0 +1,65 @@
+- name: abusech.url
+  type: group
+  description: All fields related to AbuseCH URL indicators.
+  fields:
+    #TODO for IoC expiry
+    # - name: results
+    #   type: flattened
+    - name: id
+      type: keyword
+      description: >
+        The ID of the indicator.
+
+    - name: last_online
+      type: date
+      description: >
+        Last timestamp when the URL has been serving malware.
+
+    - name: deleted_at
+      type: date
+      description: >
+        The timestamp when the indicator is (will be) deleted.
+
+    - name: urlhaus_reference
+      type: keyword
+      description: >
+        Link to URLhaus entry.
+
+    - name: url_status
+      type: keyword
+      description: >
+        The current status of the URL. Possible values are: online, offline and unknown.
+
+    - name: threat
+      type: keyword
+      description: >
+        The threat corresponding to this malware URL.
+
+    - name: reporter
+      type: keyword
+      description: >
+        The Twitter handle of the reporter that has reported this malware URL (or anonymous).
+
+    - name: larted
+      type: boolean
+      description: >
+        Indicates whether the malware URL has been reported to the hosting provider (true or false)
+
+    - name: tags
+      type: keyword
+      description: >
+        A list of tags associated with the queried malware URL
```

```suggestion
        A list of tags associated with the queried malware URL.
```
(there may be others; perhaps check in the rendered README.md)

---

updated and also resolved others in README